### PR TITLE
[Wsign-compare] warning resolutions, by directory, 7

### PR DESCRIPTION
**Pull request by tg-at-google**
Resolution affecting directories:
`tensorflow/core/`

set 1 of 3

### CODE REVIEW

**Comment by tg-at-google** [comment]
@mihaimaruseac

File: tensorflow/core/distributed_runtime/collective_rma_distributed.cc [hunk 1]
```diff
@@ -109,7 +109,8 @@ void CollectiveRemoteAccessDistributed::RecvFromPeer(
       for (const auto& chunk : extra.tensor_content()) {
         num_bytes += chunk.size();
       }
-      if (num_bytes != to_tensor->TotalBytes()) {
+      const int64 to_tensor_TotalBytes = to_tensor->TotalBytes();
+      if (num_bytes != to_tensor_TotalBytes) {
```
Comment (mihaimaruseac) [inline tensorflow/core/distributed_runtime/collective_rma_distributed.cc:113]:
```suggestion
      const int64 total_bytes = to_tensor->TotalBytes();
      if (num_bytes != total_bytes) {
```

File: tensorflow/core/distributed_runtime/rpc/grpc_remote_master.cc [hunk 2]
```diff
@@ -176,8 +176,8 @@ class GrpcRemoteMaster : public MasterInterface {
               ? deadline_with_backoff_micros
               : expired_time_micros;
       Env::Default()->SleepForMicroseconds(backoff_until - now_micros);
-      if (Env::Default()->NowMicros() > expired_time_micros &&
-          timeout_in_ms > 0) {
+      const int64 default_now_micros = Env::Default()->NowMicros();
+      if (default_now_micros > expired_time_micros && timeout_in_ms > 0) {
```
Comment (mihaimaruseac) [inline tensorflow/core/distributed_runtime/rpc/grpc_remote_master.cc:180]:
```suggestion
      const int64 now = Env::Default()->NowMicros();
      if (now > expired_time_micros && timeout_in_ms > 0) {
```

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Comment by tg-at-google** [comment]
All updates applied

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**